base-org/web
# chore: add Portuguese translation of "Base is for everyone" on the homepage 🇵🇹🇧🇷

**What changed? Why?**
Adding Portuguese to the "Base is for everyone" slider on the homepage 🇵🇹🇧🇷

**How has it been tested?**
- `yarn workspace @app/web dev`
- Open `http://localhost:3000`
- Make sure the slider is working, and the translation appears as expected
